bin/lib/c-workflow.js

```diff
@@ -13,6 +13,10 @@ const workflows = [
         description: 'Use this to initialise the project (generally after \'git clone\').',
         name: 'project-init',
```

Comment:
What about if we rename this to

```suggestion
        name: 'init',
```